clash-protocols/src/Protocols/Wishbone/Standard/Hedgehog.hs

```diff
@@ -93,6 +96,43 @@ deriving instance
   (KnownNat addressWidth, KnownNat (BitSize a), Eq a) =>
   (Eq (WishboneMasterRequest addressWidth a))
 
+{- | Checks equality for relevant parts of a 'WishboneS2M' response
+based on the corresponding 'WishboneMasterRequest'.
+For a 'Write' request, the 'readData' field is ignored, for a 'Read' request
+only the selected bytes are checked.
+>>> let readReqA = Read (0 :: BitVector 32) (0b1111 :: BitVector 4)
```

Comment:
```suggestion
{- | Checks equality for relevant parts of a 'WishboneS2M' response
based on the corresponding 'WishboneMasterRequest'.
For a 'Write' request, the 'readData' field is ignored, for a 'Read' request
only the selected bytes are checked.

>>> let readReqA = Read (0 :: BitVector 32) (0b1111 :: BitVector 4)
```

Could you make this one paragraph? Due to the newline it reads like two separate paragraphs, but it really is just two sentences..